plugin.py

```diff
@@ -90,6 +94,32 @@ def install_or_update(cls) -> None:
             raise
 
 
+class RustAnalyzerOpenDocsCommand(LspTextCommand):
+    session_name = "rust-analyzer"
+
+    def is_enabled(self) -> bool:
+        if not bool(self.session_by_name(self.session_name)):
+            return False
+
+        selection = self.view.sel()
+        if len(selection) == 0:
+            return False
+
+        return True
+
+    def run(self, edit: sublime.Edit) -> None:
+        params = text_document_position_params(self.view, self.view.sel()[0].b)
+        session = self.session_by_name(self.session_name)
+        session.send_request(Request("experimental/externalDocs", params), self.on_result)
```

Comment:
When I meant that session can be None, I meant more from type-checking point of view so checking verifying in `is_enabled` won't really help with that.

And yes, it's a hypothetical issue but types are not forgiving :)